### PR TITLE
Added apoc.date.convert() and apoc.date.add()

### PR DESCRIPTION
**Pull request by InverseFalcon**
This fulfills the feature request in #289.

### CODE REVIEW

File: src/main/java/apoc/date/Date.java [hunk 1]
```diff
@@ -136,6 +136,19 @@ public String systemTimezone() {
 		return TimeZone.getDefault().getID();
 	}
 
+	@UserFunction
+	@Description("apoc.date.convert(12345,'ms|s|m|h|d','ms|s|m|h|d') convert a time unit into a different time unit")
+	public Long convert(@Name("time") long time, @Name(value = "unit") String unit, @Name(value = "toUnit") String toUnit) {
```
Comment (jexp) [inline src/main/java/apoc/date/Date.java]: are there any sensible default values?

**Comment by jexp** [comment]
looks good to me, can you add them to docs/overview.adoc too?

File: src/main/java/apoc/date/Date.java (hunk 1 above)
Comment (InverseFalcon) [inline src/main/java/apoc/date/Date.java]: Maybe 'ms' for the `unit` value, but I'm wondering if in this case a default value would be easier to work with or make usage more error-prone.

Comment (jexp) [inline src/main/java/apoc/date/Date.java]: Probably not worth it then.

**Pull request merged**